third_party/grpc/compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -1069,9 +1069,8 @@ static void PrintService(const ServiceDescriptor* service,
   GrpcWriteServiceDocComment(p, service);
   p->Print(
       *vars,
-      "@$Generated$(\n"
-      "    value = \"by gRPC proto compiler$grpc_version$\",\n"
-      "    comments = \"Source: $file_name$\")\n"
+      "// Generated by gRPC proto compiler$grpc_version$\n"
+      "// Source: $file_name$\n"
```

Comment:
This doesn't belong in this PR, it should be a separate change.

Reply:
Done.